Corona-Warn-App/src/main/res/layout/fragment_family_test_consent.xml

```diff
@@ -250,4 +250,15 @@
 
     <include layout="@layout/merge_guidelines_side" />
 
+    <com.google.android.material.progressindicator.CircularProgressIndicator
```

Comment:
remove this and change the button? -> ProgressLoadingButton to beconsistent with the ui

Reply:
I tried but realised that ProgressLoadingButton doesn't really support being disabled reliably.
As ProgressLoadingButton appears on too many places this should be changed in an Submit button refactoring ticket

Reply:
Changing the property 'isLoading' of ProgressLoadingButton does not work?

Reply:
Have you tried disabling the ProgressLoadingButton.defaultButton?
I agree that we should refactor the button as it has other issues

Reply:
Yes, I tried, but there were some style inconsistencies and also isClickable interfered, which makes it necessary to do a ProgressLoadingButton refactoring if we want to fully support disabling